xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAEStream.cpp

```diff
@@ -671,17 +710,34 @@ bool CActiveAEStreamBuffers::IsDrained()
 
 void CActiveAEStreamBuffers::SetRR(double rr)
 {
-  m_resampleBuffers->SetRR(rr);
+  if (rr == 1.0)
```

Comment:
Isn't this block redundant? Looks like the next if block will just do the same calls again.